frontend/src/components/activity/content/ColumnLayout.vue

```diff
@@ -98,6 +122,9 @@ export default {
     isRoot() {
       return this.contentNode._meta.self === this.contentNode.root()._meta.self
     },
+    isDefaultVariant() {
+      return this.clientWidth > 870
```

Comment:
As per discussion, change this so it respects the number of columns, e.g.
```suggestion
      return this.clientWidth / this.numColumns > 260
```